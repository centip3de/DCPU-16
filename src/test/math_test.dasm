SET A 0x30
SUB A 1

SET B 0x30
ADD B 1

SET C 0x20
SET X 0x50
SUB X [C]

SET Y 0x10
MOD Y 0x10
