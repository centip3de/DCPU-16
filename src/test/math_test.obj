0x7c01 0x30 0x8403 0x7c11 0x30 0x8412 0x7c21 0x20 0x7c31 0x50 0x2833 0xc041 0xc046