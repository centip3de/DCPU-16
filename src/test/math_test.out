********* REG DUMP *********
A: 0x2f
B: 0x31
C: 0x20
X: 0x50
Y: 0x0
Z: 0x0
I: 0x0
J: 0x0
PC: 0xd
O: 0x0
SP: 0xffff
